packages/ui-react/src/components/TileDock/TileDock.tsx

```diff
@@ -263,13 +268,17 @@ function TileDock<T>({
             />
           ) : null}
           {tileList.map((tile: Tile<T>, listIndex) => {
+            const posInSet = items.findIndex((item) => item === tile.item); // TODO optimize this for performances
             const isInView = !isMultiPage || (listIndex > tilesToShow - slideOffset && listIndex < tilesToShow * 2 + 1 - slideOffset);
 
             return (
               <li
                 key={tile.key}
+                aria-setsize={items.length}
+                aria-posinset={posInSet + 1}
```

Comment:
These are needed to let the screen reader know on which position you are in the list. This is needed because `aria-hidden="true"` is applied, causing the screen reader to not take them in to consideration.